couchbase/json-queries.yml

```diff
@@ -0,0 +1,151 @@
+type: json
+data:
+  url: /admin/vitals
+  server: ${COUCHBASE_QUERY_HOSTPORT}
+  basicHttpAuthUsername: ${COUCHBASE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${COUCHBASE_BASICAUTH_PASSWORD}
+
+observation:
+  - name: queries
+    metricNamespace: couchbase
+    path: $.
+
+    metric:
+      - name: cores
+        source: cores
+        type: long_gauge
+        label: cores
+        description: Cores
+
+      - name: cpu.sys.percent
+        source: cpu.sys.percent
+        type: long_gauge
+        label: system cpu
+        unit: "%"
+        description: Percentage of CPU used by the system
+
+      - name: cpu.user.percent
+        source: cpu.user.percent
+        type: long_gauge
+        label: user cpu
+        unit: "%"
+        description: Percentage of CPU used by the user
+
+      - name: gc.num
+        source: gc.num
+        type: long_gauge
+        label: gc num
+        description: Number of objects garbage collected
+
+      - name: gc.pause.percent
+        source: gc.pause.percent
+        type: long_gauge
+        label: gc pause percent
+        unit: "%"
+        description: Garbage collection pause percentage
+
+      - name: gc.pause.time
+        source: gc.pause.time
+        type: long_gauge
+        label: gc pause time
+        unit: seconds
+        description: Garbage collection pause time
+
+      - name: memory.system
```

Comment:
Well, there are two memory metrics related to couchbase server itself. We could say that STA will collect them as it is also monitoring processes, but there is no STA in public AA repo (plus it may be interesting to see if what STA collects differs from couchbase metrics).

Reply:
> there is no STA in public AA repo 

But then the same holds true for CPU metrics I comments on above, no?

> there are two memory metrics related to couchbase server itself.

Oh!  If that is the case, then ignore my comment here and let's keep the memory.xxxx metrics.  I thought this was just Couchbase exposing the system memory metrics.

Reply:
> same holds true for CPU metrics

true, not sure what to do about these.